src/prefect/infrastructure/kubernetes.py

```diff
@@ -75,11 +75,11 @@ class KubernetesJob(Infrastructure):
         default="kubernetes-job", description="The type of infrastructure."
     )
     # shortcuts for the most common user-serviceable settings
-    image: str = Field(
+    image: Optional[str] = Field(
         default_factory=get_prefect_image_name,
         description="The tag of a Docker image to use for the job. Defaults to the Prefect image.",
     )
-    namespace: str = Field(
+    namespace: Optional[str] = Field(
         default="default", description="The Kubernetes namespace to use for this job."
```

Comment:
```suggestion
        default="default", description="The Kubernetes namespace to use for this job.  Use `None` if you will be customizing the `job` and already define the `namespace` in your manifest"
```

Reply:
Not taken verbatim, but addressed in https://github.com/PrefectHQ/prefect/pull/7244/commits/cd7629b06bca087576371610f6d4ef537974ace9